src/MainWindow.vala

```diff
@@ -46,6 +48,8 @@ public class Mail.MainWindow : Gtk.ApplicationWindow {
         {ACTION_REPLY,              on_reply             },
         {ACTION_REPLY_ALL,          on_reply_all         },
         {ACTION_FORWARD,            on_forward           },
+        {ACTION_MARK_READ, on_mark_read },
+        {ACTION_MARK_UNREAD, on_mark_unread },
```

Comment:
can you keep this align as above

Reply:
Yes, but I feel like we should probably not try to do this kind of alignment since it just causes more line changes in the future if we ever update this